### PR TITLE
Feature/test irods 4.2.7 clients

### PR DESCRIPTION
**Pull request by kjsanger**
Test using new iRODS 4.2.7 clients.

Fix data object replacement checksum behaviour in line with test.

WTSI to WSI to update for Institute rebranding.

### CODE REVIEW

File: lib/WTSI/NPG/iRODS.pm [hunk 1]
```diff
@@ -1442,7 +1442,7 @@ sub _replace_object {
       $self->logconfess("Invalid checksum action '$checksum_action'");
   }
   else {
-    $checksum_action = $CALC_CHECKSUM;
+    $checksum_action = $SKIP_CHECKSUM;
```
Comment (mgcam) [inline lib/WTSI/NPG/iRODS.pm:1445]: This changes the default action, which previously was to calculate the ckecksum and is still documented as such. Will the change of default affect existing code?

Comment (dkj) [reply lib/WTSI/NPG/iRODS.pm:1445]: This caught my eye too and was the change I brought up in Wednesday's meeting: I think @kjsanger described it as a real bug which had been hidden in a SKIP in the tests until now.

Comment (mgcam) [reply lib/WTSI/NPG/iRODS.pm:1445]: We now have time to test and fix anything downstream if needed. The POD for this function has to reflect the default change.

**Comment by kjsanger** [comment]
`add_object` is used in 6 location by some tests in `npg_irods`, but nowhere else. I can test those to see if setting `$SKIP_CHECKSUM` causes a failure

**Comment by mgcam** [comment]
> `add_object` is used directly in 6 location by some tests in `npg_irods`, but nowhere else. I can test those to see if setting `$SKIP_CHECKSUM` causes a failure
I'll merge this pr once the Travis tests pass and then we will deal with any failures, which will appear in Jenkins' full stack build

**Pull request merged**